src/infra/logger/types/logger.types.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
I think this file is not in use

Reply:
That's correct! Fixed by removing the file.